src/lib/client/store/activities.ts

```diff
@@ -1,10 +1,15 @@
-import { createSlice } from '@reduxjs/toolkit';
+import { CaseReducer, createSlice } from '@reduxjs/toolkit';
+import { TypedActionCreator } from '@reduxjs/toolkit/dist/mapBuilders';
 
 import { ActivityActions, ActivityState, BasicReducers } from '@respond/lib/state';
 import { Activity, isActive as isParticipantStatusActive, ParticipantStatus, ParticipantUpdate } from '@respond/types/activity';
 
 import { RootState } from '.';
 
+export interface ReducerBuilderStub {
+  addCase<ActionCreator extends TypedActionCreator<string>>(actionCreator: ActionCreator, reducer: CaseReducer<ActivityState, ReturnType<ActionCreator>>): ReducerBuilderStub;
+}
```

Comment:
Instead of creating a new type here, can't you just use the built-in type for the test builder: ActionReducerMapBuilder<ActivityState>

Or did you do this to be more explicit about which methods we're actually using from the main builder type?

Reply:
I wanted the interface so I could provide a stub in the test case. I didn't want to implement all the methods on `ActionReducerMapBuilder`, so I just pulled out the one method we're using. Good question, I hadn't actually verified that `ActionReducerMapBuilder` contained more than the one method.